common/metrics/defs.go

```diff
@@ -494,7 +507,14 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		CadenceErrShardOwnershipLostCounter:       {metricName: "cadence.errors.shard-ownership-lost", metricType: Counter},
 		CadenceErrEventAlreadyStartedCounter:      {metricName: "cadence.errors.event-already-started", metricType: Counter},
 	},
-	Matching: {},
+	Matching: {
+		PollSuccessCounter:          {metricName: "poll.success"},
```

Comment:
we do have separate counters {PollForActivity, PollForDecisionTask} at the top level (i.e. matchingHandler). My understanding is that, at the taskMgr level, we don't differentiate between these two tasks. Yes/No ?